CONTRIBUTING.md

```diff
@@ -1,6 +1,30 @@
 
 # Contributing guidelines
 
+## Docker for refinebio-examples
+
+This repository uses a Docker image to manage dependencies and keep software versions consistent.
+Development should take place in the Docker container.
+
+### Setting up the docker container
+
+You can build the docker image locally using:
+```
+docker build -< Dockerfile -t refinebio-examples
+```
+Or by pulling the image from Docker hub.
+```
+docker pull ccdl/refinebio-examples
+```
+
+Navigate to your local `refinebio-examples` repository.
```

Comment:
This should happen before they try to `docker build` if they are going that route. Also, if you are there, then you can use:
```
docker run --mount type=bind,target=/home/rstudio,source=$PWD -e PASSWORD=<PASSWORD> -p 8787:8787 refinebio-examples
```
to provide less that needs to be replaced.